### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by siburu**

### CODE REVIEW

**Comment by siburu** [comment]
@bluele I confirmed this CI works well by using my account.
https://github.com/siburu/cosmos-ethereum-ibc-lcp/actions/runs/10846314319/job/30099015991

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: test
+on: [push, pull_request]
+
+jobs:
+  e2e-test:
+    runs-on: ubuntu-24.04
```
Comment (bluele) [inline .github/workflows/test.yml:6]: @siburu  Can you use `ubuntu-22.04` same as lcp and lcp-go?https://github.com/datachainlab/lcp/blob/main/.github/workflows/test.yml
(Also, note that we currently use the intel sgx sdk is for 22.04 in the test)

Comment (siburu) [reply .github/workflows/test.yml]: @bluele yes, done!

**Review by bluele — APPROVED** [review]
@siburu Thanks! LGTM👍

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        run: |
+          curl -LO https://download.01.org/intel-sgx/sgx-linux/2.19/distro/ubuntu22.04-server/sgx_linux_x64_sdk_2.19.100.3.bin
+          chmod +x ./sgx_linux_x64_sdk_2.19.100.3.bin
+          echo -e 'no\n/opt' | ./sgx_linux_x64_sdk_2.19.100.3.bin
+      - name: Run E2E test
+        run: |
+          source /opt/sgxsdk/environment
+          make prepare-contracts
+          make build-images
+          make USE_UPGRADE_TEST=no e2e-test
```
Comment (bluele) [inline .github/workflows/test.yml:35]: @siburu Is this correct? I think we should set `USE_UPGRADE_TEST` to `yes` in CI.

**Pull request merged**